### ISSUE TITLE
Improve message on mbedls when serial port is not found

### ISSUE THREAD

**Issue by mlnx** [ARMmbed/greentea#149]
I get this message when trying to run tests on my platform:

```
mbedgt: test specification file 'C:\mbed\test_K64\mbed-os-program\.build/tests\K64F\GCC_ARM\test_spec.json' (specified with --test-spec option)
mbedgt: using 'C:\mbed\test_K64\mbed-os-program\.build/tests\K64F\GCC_ARM\test_spec.json' from current directory!
mbedgt: detecting connected mbed-enabled devices...
mbedgt: detected 1 device
mbedgt: can't detect all properties of the device!
        property 'serial_port' is 'None'    <-----
mbedgt: processing target 'K64F' toolchain 'GCC_ARM' compatible platforms...
        use 0 instances for testing
mbedgt: all tests finished!
mbedgt: shuffle seed: 0.6392864606
mbedgt: no platform/target matching tests were found!
mbedgt: no target matching platforms were found!
mbedgt: completed in 0.08 sec
mbedgt: exited with code -110
[mbed ERROR] "mbedgt" returned error code -110.   <----
[mbed ERROR] Command "mbedgt --test-spec C:\mbed\test_K64\mbed-os-program\.build/tests\K64F\GCC_ARM\test_spec.json" in "C:\mbed\test_K64\mbed-os-program"
---
```

It's not obvious there this something wrong by looking at error -110 or this message ``` property 'serial_port' is 'None'```

Could you please display a message similar to this: "Please check your serial port driver is correctly installed" or something like that. Thanks.

cc @PrzemekWirkus

**Comment by ciarmcom** [comment]
ARM Internal Ref: IOTSFW-2725

**Comment by PrzemekWirkus** [comment]
This issue is addressed in #152.
